pydriller/domain/commit.py

```diff
@@ -562,6 +590,161 @@ def _get_branches(self):
             branches.add(branch.strip().replace('* ', ''))
         return branches
 
+
+    _UNIT_SIZE_LOW_RISK_THRESHOLD = 15
+    """
+    Threshold used in the Delta Maintainability Model to establish whether a method
+    is low risk in terms of its size.
+    """
+    @staticmethod
+    def _low_risk_unit_size(method) -> bool:
+        """
+        Predicate indicating whether a method is low risk in terms of its size,
+        i.e., whether its number of lines of code remains on or below the low risk threshold.
+        """
+        return method.nloc <= Commit._UNIT_SIZE_LOW_RISK_THRESHOLD
```

Comment:
What if we bring this to the Method class? And we make it public. As well as `UNIT_SIZE_LOW_RISK_THRESHOLD`

Reply:
So a user can see if a method is "low_risk_unit_size" or "high_risk_unit_size"